core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataTest.scala

```diff
@@ -307,6 +310,41 @@ class GroupMetadataTest {
     assertEquals(Some(offset), group.offset(partition))
   }
 
+  @Test
+  def testStableGroupChooseExpiredOffsets(): Unit = {
```

Comment:
It's better to make a meta comment for the purpose of this test, or augment the test case name to `testStableGroupOnlyChooseUnsubscribedExpiredOffsets`